sim/adversary/adversary.go

```diff
@@ -13,7 +13,8 @@ type Host interface {
 	// Sends a message to all other participants, immediately.
 	// Note that the adversary can subsequently delay delivery to some participants,
 	// before messages are actually received.
-	BroadcastSynchronous(sender gpbft.ActorID, msg gpbft.GMessage)
+	// The sync flag can be used to determine how the message is broadcast.
+	Broadcast(sender gpbft.ActorID, msg *gpbft.GMessage, synchronouos bool)
```

Comment:
It's now much less obvious at the call sites when the special adversary powers are being invoked. I would prefer this method were named specially when the synchronous behaviour is invoked, and adversaries that aren't using it just use the same method as the normal participant.

Reply:
I see, I went with this approach per this comment: https://github.com/filecoin-project/go-f3/pull/188/files#r1595990496 but this makes sense. I'll revert to how it was before.